test/unit/taxonomy_prototype/taxon_finder_test.rb

```diff
@@ -0,0 +1,24 @@
+require "test_helper"
+require "taxonomy_prototype/taxon_finder"
+
+describe TaxonomyPrototype::TaxonFinder do
+  describe "find_by(slug:)" do
+    before do
+      File.stubs(:exist?).returns(true)
```

Comment:
reckon in-lining the File stub might be clearer, the CSV stuff is needed by both existing tests. Will also introduce a third test for the scenario where no file is present.